WebView/app/src/main/assets/main.js

```diff
@@ -14,26 +14,18 @@
  * limitations under the License.
  */
 function sendAndroidMessage() {
-	Weather.sendMessage("The weather in WebView City today is: " + document.getElementById("description").innerHTML);
+	Weather.sendMessage(`The weather in ${document.getElementById("title").innerText} today is
+	${document.getElementById("description").innerText} `);
```

Comment:
This will add `\n\t` in the middle of the string. Could we instead concatenate the strings?